src/data/queries/banners.ts

```diff
@@ -11,57 +11,58 @@ export type BannerDataOpts = {
 // A given page does not reference a banner node via entity reference, banner node types have a field that lists what
 // paths they are supposed to be visible on. This endpoint queries banners based on their path lists.
 // See docroot/modules/custom/va_gov_api/src/Resources/BannerAlerts.php in va.gov-cms for more info.
```

Comment:
Might want to update the reference to the code that provides the banner response.